cli/src/main/resources/nls/Ide.properties

```diff
@@ -2,28 +2,32 @@ usage=Usage:
 values=Values:
 commandlets=Available commandlets:
 options=Options:
-cmd-az=Tool commandlet for Azure CLI.
+cmd---version=Print the version of IDEasy.
+cmd-az=Tool commandlet for Azure CLI
+cmd-complete=Internal commandlet for bash auto-completion
+cmd-eclipse=Tool commandlet for Eclipse (IDE)
 cmd-env=Print the environment variables to set and export.
 cmd-get-version=Get the version of the selected tool.
-cmd-gh=Tool commandlet for Github CLI.
+cmd-gh=Tool commandlet for Github CLI
 cmd-gradle=Tool commandlet for Gradle (Build-Tool)
 cmd-helm=Tool commandlet for Helm (Kubernetes Package Manager)
 cmd-help=Prints this help.
 cmd-install=Install the selected tool.
 cmd-java=Tool commandlet for Java (OpenJDK)
-cmd-kotlinc=Tool commandlet for Kotlin.
-cmd-kotlincnative=Tool commandlet for Kotlin-Native.
+cmd-kotlinc=Tool commandlet for Kotlin
+cmd-kotlincnative=Tool commandlet for Kotlin-Native
 cmd-list-version=List the available versions of the selected tool.
 cmd-mvn=Tool commandlet for Maven (Build-Tool)
 cmd-node=Tool commandlet for Node.js (JavaScript runtime)
 cmd-oc=Tool commandlet for Openshift CLI (Kubernetes Management Tool)
 cmd-quarkus=Tool commandlet for Quarkus (Framework for cloud-native apps)
 cmd-set-version=Set the version of the selected tool.
-cmd-terraform=Tool commandlet for Terraform.
+cmd-shell=Commandlet to start build-in shell with advanced auto-completion.
```

Comment:
```suggestion
cmd-shell=Commandlet to start built-in shell with advanced auto-completion.
```

Reply:
Indeed a typo:
https://whichiscorrect.com/build-in-vs-built-in/